### ISSUE TITLE
Form localstorage does not store select menus, checkboxes, or other input types

### ISSUE THREAD

**Issue by chrisblakley** [chrisblakley/Nebula#1657]
Checkboxes, selects, date, and other non-text input types are not captured by the Nebula localstorage function.

**Comment by chrisblakley** [comment]
Maybe its fine to ignore "acceptance" style checkboxes.

**Comment by chrisblakley** [comment]
I may table this issue in favor of upcoming KV Storage: https://github.com/chrisblakley/Nebula/issues/1832

**Comment by patphg** [comment]
Looking into this. Is there a tricky reason you never added more input types?

**Comment by chrisblakley** [comment]
Not necessarily– just low priority as far as use cases go. They'll need their own event listeners and can't use the same "filler" as-is since it currently just works for strings.

**Comment by patphg** [comment]
https://github.com/chrisblakley/Nebula/pull/2098
